README.md

```diff
@@ -1 +1,4 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+# description
+
```

Comment:
- [ ] Kindly be noticed that the README file should be descriptive? AKA: Microverse README template should be used.